tests/unittests/test_null_cell_format.py

```diff
@@ -61,28 +61,6 @@ def test_null_datetime_effectiveFormat(self):
         # verify the returned schema has expected field types and format
         self.assertIn(expected_format,returned_formats)
 
-    def test_null_date_effectiveFormat(self):
-        """
-        Test when no value is given in second row of date, discovery is locking at 'effectiveFormat'.
-        And returns type and format in schema according to that.
-        """
-        sheet = SHEET
-        sheet["data"][0]["rowData"][1]["values"][0]["effectiveFormat"]["numberFormat"]["type"] = "DATE"
-
-        expected_format = {
-                            "type": [
-                                "null",
-                                "string"
-                            ],
-                            "format": "date"
-                        }
-
-        sheet_json_schema, columns = schema.get_sheet_schema_columns(sheet)
-        returned_formats = sheet_json_schema["properties"]["Column1"]["anyOf"]
-
-        # verify the returned schema has expected field types and format
-        self.assertIn(expected_format,returned_formats)
-
```

Comment:
I think there is still value in this test as the format is different from the above test.